configure.sh

```diff
@@ -23,147 +23,58 @@ function write_to_bazelrc() {
-      echo 'Already have tensorflow non-gpu installed. Uninstalling......\n'
-      pip uninstall tensorflow
-    elif [[ $(pip show tf-nightly) == *tf-nightly* ]]; then
-      echo 'Already have tensorflow non-gpu installed. Uninstalling......\n'
-      pip uninstall tf-nightly
-    fi
-    # Install CPU version
-    if [[ "$PIP_MANYLINUX2010" == "0" ]]; then
-        echo 'Installing tensorflow-gpu 1.14.....\n'
-        pip install tensorflow-gpu==1.14
-    else
-        echo 'Installing tensorflow-gpu 2.....\n'
-        pip install tensorflow-gpu
-    fi
-  fi
+  while [[ "$PIP_MANYLINUX2010" == "" ]]; do
+    read -p "Does the pip package have tag manylinux2010 (usually the case for nightly release after Aug 1, 2019, or official releases past 1.14.0)?. Y or enter for manylinux2010, N for manylinux1. [Y/n] " INPUT
+    case $INPUT in
+      [Yy]* ) PIP_MANYLINUX2010=1;;
+      [Nn]* ) PIP_MANYLINUX2010=0;;
+      "" ) PIP_MANYLINUX2010=1;;
+      * ) echo "Invalid selection: " $INPUT;;
+    esac
+  done
 fi
 
-
-TF_CFLAGS=( $(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_compile_flags()))') )
-TF_LFLAGS="$(python -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_link_flags()))')"
-
-write_to_bazelrc "build:cuda --define=using_cuda=true --define=using_cuda_nvcc=true"
-if [[ "$PIP_MANYLINUX2010" == "0" ]]; then
-  write_to_bazelrc "build:cuda --crosstool_top=@local_config_cuda//crosstool:toolchain"
-fi
 if is_linux; then
   write_to_bazelrc "build:manylinux2010 --crosstool_top=//third_party/toolchains/preconfig/ubuntu16.04/gcc7_manylinux2010-nvcc-cuda10.0:toolchain"
 fi
 write_to_bazelrc "build --spawn_strategy=standalone"
 write_to_bazelrc "build --strategy=Genrule=standalone"
 write_to_bazelrc "build -c opt"
 
-
-write_action_env_to_bazelrc "TF_HEADER_DIR" ${TF_CFLAGS:2}
-SHARED_LIBRARY_DIR=${TF_LFLAGS:2}
-SHARED_LIBRARY_NAME=$(echo $TF_LFLAGS | rev | cut -d":" -f1 | rev)
-if ! [[ $TF_LFLAGS =~ .*:.* ]]; then
-  if [[ "$(uname)" == "Darwin" ]]; then
-    SHARED_LIBRARY_NAME="libtensorflow_framework.dylib"
-  else
-    SHARED_LIBRARY_NAME="libtensorflow_framework.so"
-  fi
-fi
-write_action_env_to_bazelrc "TF_SHARED_LIBRARY_DIR" ${SHARED_LIBRARY_DIR}
-write_action_env_to_bazelrc "TF_SHARED_LIBRARY_NAME" ${SHARED_LIBRARY_NAME}
-write_action_env_to_bazelrc "TF_NEED_CUDA" ${TF_NEED_CUDA}
-
-# TODO(yifeif): do not hardcode path
-if [[ "$TF_NEED_CUDA" == "1" ]]; then
-  write_action_env_to_bazelrc "CUDNN_INSTALL_PATH" "/usr/lib/x86_64-linux-gnu"
-  write_action_env_to_bazelrc "TF_CUDA_VERSION" "10.0"
-  write_action_env_to_bazelrc "TF_CUDNN_VERSION" "7"
-  write_action_env_to_bazelrc "CUDA_TOOLKIT_PATH" "/usr/local/cuda"
-  write_to_bazelrc "build --config=cuda"
-  write_to_bazelrc "test --config=cuda"
-fi
-
-
 if [[ "$PIP_MANYLINUX2010" == "1" ]]; then
   if is_linux; then
     write_to_bazelrc "build --config=manylinux2010"
     write_to_bazelrc "test --config=manylinux2010"
   fi
+fi
+
+if ! is_windows; then
```

Comment:
I spend ~10min trying to compile it on Windows, but without look. Though these changes should at least make the configure script Windows compatible.